src/build_multiversion.zig

```diff
@@ -5,7 +5,7 @@ const builtin = @import("builtin");
 const assert = std.debug.assert;
 
 const multiversioning = @import("./multiversioning.zig");
-const flags = @import("flags.zig");
```

Comment:
Here and elsewhere, let's rather do `const stdx = @import("stdx")`, and then `stdx.flags`. 

Given that we use `flags` infrequently, qualified name feels like a better fit.